package/MDAnalysis/core/groups.py

```diff
@@ -197,15 +198,66 @@ def n_segments(self):
         return len(self.segments)
 
 
-class AtomGroupBase(object):
-    """AtomGroup base class.
+class AtomGroup(object):
```

Comment:
With these, maybe they should be `_AtomGroup` just so people don't find them and use them, because they won't work as they're not the patched up versions